tools/docker/builder/openwrt/scripts/build-openwrt.sh

```diff
@@ -34,6 +35,23 @@ case $TARGET_DEVICE in
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y"
             echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y"
             echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y"
+            # AMBIORIX modules to work with u-bus
+            echo "CONFIG_PACKAGE_amxb-ubus=y"
+            # AMBIORIX applications for debug
+            echo "CONFIG_PACKAGE_amxb-inspect=y"
+            echo "CONFIG_PACKAGE_amxo-cg=y"
+            echo "CONFIG_PACKAGE_amxrt=y"
+            # AMBIORIX libs
+            echo "CONFIG_PACKAGE_libamxb=y"
+            echo "CONFIG_PACKAGE_libamxc=y"
+            echo "CONFIG_PACKAGE_libamxd=y"
+            echo "CONFIG_PACKAGE_libamxj=y"
+            echo "CONFIG_PACKAGE_libamxo=y"
+            echo "CONFIG_PACKAGE_libamxp=y"
+            echo "CONFIG_PACKAGE_libevent2=y"
+            # Dependencies of AMBIORIX
+            echo "CONFIG_PACKAGE_uriparser=y"
+            echo "CONFIG_PACKAGE_yajl=y"
```

Comment:
The different `CONFIG_TARGET...` and the config line for prplmesh are convenient to have here because they depend on arguments.
For other stuff we want to add however, it would be easier to have a file containing the configuration directly (no need for the echo lines, you could directly copy and paste from the `.config`).
Then here it would just be a matter of copying the partial config file to `.config`.
This is something we did before to build different targets with different GCC versions, but it was reverted at some point because it wasn't used anymore.
You can check the following commit if you want an example of what I'm talking about: https://github.com/prplfoundation/prplMesh/blob/ba02ad94f6d6129b8f5b10fdd619d76e40ab1948/tools/docker/builder/openwrt/Dockerfile
(specifically this line: `COPY --chown=openwrt:openwrt configs/base_${BASE_CONFIG}_config /home/openwrt/base_config`)

I would understand if you don't want to do this as part of this PR though, so (nitpick)